fluent-plugin-google-cloud.gemspec

```diff
@@ -19,6 +19,7 @@ eos
   gem.test_files    = gem.files.grep(/^(test)/)
   gem.require_paths = ['lib']
 
+  gem.add_runtime_dependency 'chronic_duration', '~> 0.10.6'
```

Comment:
I would ask the Google Ruby libraries folks. This is a common issue with all of our APIs, and should have a common solution by now.